toolbox/load_aware.py

```diff
@@ -72,4 +72,8 @@ def scale_test(self, distribution="poisson", duration=60.0, instances=10, namesp
             print(f"Can't run scale test using unknown distribution: {distribution}")
             sys.exit(1)
 
+        if scheduler not in ("default", "trimaran"):
+            print(f"Can't run scale test with unknown scheduler: {scheduler}")
+            sys.exit(1)
+
```

Comment:
is there a way to check if the value of `scheduler` is correct?
something like this?
```
oc get scheduler/{scheduler}
```
if yes, then this could be at the top of the Ansible task file and removed from here

otherwise, never mind, we won't test it with another secondary scheduler than Trimaran anytime soon :D

Reply:
I'll have to give that a shot. It didn't occur to me that the scheduler itself was a resource.

Reply:
```
oc get schedulers -oyaml
apiVersion: v1
items:
- apiVersion: config.openshift.io/v1
  kind: Scheduler
  metadata:
    name: cluster
  spec:
    mastersSchedulable: false
    policy:
      name: ""
  status: {}
kind: List
metadata:
  resourceVersion: ""
```
we'll see if you get more than one `scheduler` :)

Reply:
Looks like I'm only seeing the one with Trimaran deployed. I don't think the specific scheduler gets listed as a `scheduler` resource.